sdk/communication/AzureCommunicationChat/Source/Generated/ChatThread.swift

```diff
@@ -32,7 +32,7 @@ public final class ChatThreadOperation {
     public func listChatReadReceipts(
         chatThreadId: String,
         withOptions options: ListChatReadReceiptsOptions? = nil,
-        completionHandler: @escaping HTTPResultHandler<PagedCollection<ChatMessageReadReceipt>>
+        completionHandler: @escaping HTTPResultHandler<PagedCollection<ChatMessageReadReceiptInternal>>
```

Comment:
Related to your comment on "internal" models being exposed. The generated code returns the internal model & the convenience layer converts it to the model thats supposed to be exposed to the user. 

The user should not be calling this generated method directly though, I will try the changes in autorest.swift again though hopefully that can address some of these issues

Reply:
Okay, that makes sense. However, it does seem like I will need to also proceed with making the clients internal, especially since the method names in the convenience layer don't match the ones in the generated client.

Reply:
Left details on autorest.swift PR, I am still having some issues applying the changes properly